ambry-store/src/main/java/com/github/ambry/store/BlobStoreCompactor.java

```diff
@@ -557,6 +580,16 @@ private boolean copyDataByIndexSegment(LogSegment logSegmentToCopy, IndexSegment
     return copiedAll;
   }
 
+  /**
+   * Determines if {@code copyCandidate} container in the status of DELETED_IN_PROGRESS or INACTIVE.
+   * @param copyCandidate the {@link IndexEntry} to check
+   * @param selectedContainerPairSet the collection of AccountId and ContainerId pairs in the given status.
+   */
+  private boolean isFromDeletedContainer(IndexEntry copyCandidate, Set<Pair<Short,Short>> selectedContainerPairSet) {
```

Comment:
selectedContainerPairSet is a class member, do we need to pass it as argument?

Reply:
Updated.